constants/meetups.ts

```diff
@@ -79,4 +79,10 @@ Ready to level up? See you at Leetcode Night!A combination of the YEGRB, Exchang
     to: 'https://hacked.compeclub.com/',
     description: 'One of Alberta\'s Largest Student Run Hackathons. HackED is the premiere annual 24-hour hackathon from the Computer Engineering Club of the University of Alberta.',
   },
+  {
+    name: 'Edmonton WordPress Meetup',
+    image: '/meetups/wordpress.png?updatedAt=1728262345205',
```

Comment:
Sorry! I gave you a link with a query param! You can remove it and then I'll merge this!
```suggestion
    image: '/meetups/wordpress.png',
```